dspace/config/dspace.cfg

```diff
@@ -1675,3 +1675,10 @@ include = ${module_dir}/versioning.cfg
 include = ${module_dir}/workflow.cfg
 include = ${module_dir}/external-providers.cfg
 include = ${module_dir}/ldn.cfg
+
+#---------------------------------------------------------------#
+#---------------PERSONAL DATA STORED IN PROVENANCE--------------#
+#---------------------------------------------------------------#
+#Provenance policy
+#disabled by default
+metadata.privacy.dc.description.provenance = false
```

Comment:
@AndrewAlesik : It appears you misunderstood my request here.  Sorry, I wasn't clear, as I wanted you to keep most of your configuration the exact same as before & just move it to `dspace.cfg`.  So, this should be updated to say something like this:
```
# Behavior to generate provenance message, if set true,
# personal data (e.g. email) of submitter will be hidden in dc.description.provenance
# Disabled by default.  To enable, uncomment the next line
# metadata.privacy.dc.description.provenance = true
```

I'd also recommend moving this entire section up next to the "metadata.hide" settings.  As that ensures these similar settings are next to each other. So, move all your new configuration lines up to line 976 (right after the `metadata.hide` settings):
https://github.com/DSpace/DSpace/blob/main/dspace/config/dspace.cfg#L976

Reply:
Hi @tdonohue , 
I've moved these changes to the dspace.cfg, renamed the property, and updated the documentation. Could you approve the changes?